tests/analysis_tests/tests/src/expected/Jsx2.res.txt

```diff
@@ -523,28 +517,8 @@ Path Nested.
   }]
 
 Hover src/Jsx2.res 162:12
-Nothing at that position. Now trying to use completion.
-posCursor:[162:12] posNoWhite:[162:11] Found expr:[162:3->162:21]
-posCursor:[162:12] posNoWhite:[162:11] Found expr:[162:6->162:21]
-posCursor:[162:12] posNoWhite:[162:11] Found expr:[162:6->162:20]
-JSX <Comp:[162:6->162:10] age[162:11->162:14]=...[162:15->162:17]> _children:162:18
-Completable: Cjsx([Comp], age, [age])
-Package opens Stdlib.place holder Pervasives.JsxModules.place holder
-Resolved opens 1 Stdlib
-Path Comp.make
-{"contents": {"kind": "markdown", "value": "```rescript\nint\n```"}}
+{"contents": {"kind": "markdown", "value": "```rescript\nComp.props<int>\n```\n\n---\n\n```\n \n```\n```rescript\ntype Comp.props<'age> = {age: 'age}\n```\nGo to: [Type definition](command:rescript-vscode.go_to_location?%5B%22Jsx2.res%22%2C157%2C2%5D)\n"}}
```

Comment:
was this incorrect before?

Reply:
Not really, but after these changes there was type information for this hover, where previously there wasn't. I suspect this is the case because of more accurate ranges in the node now.